packages/kit/src/views/DAppConnection/hooks/useDappAccountSwitch.ts

```
@@ -236,6 +240,49 @@ export function useDappAccountSwitch({
+            homeAccountSelectorInfo: selectedAccount,
+            connectedAccountInfo: result.connectedAccountsInfo[0],
+          });
+        const isSyncing =
+          await backgroundApiProxy.serviceDApp.getAlignPrimaryAccountProcessing();
+
+        if (!isSameAccount && !isSyncing) {
+          const scopes = getScopeFromImpl({
+            impl: result.connectedAccountsInfo[0].networkImpl,
+          });
+          // update connected accounts info from home
+          if (scopes && scopes.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)